FreeRTOS_DNS.c

```diff
@@ -378,7 +378,7 @@
     #include "pack_struct_end.h"
     typedef struct xDNSAnswerRecord DNSAnswerRecord_t;
 /** @brief this is for debugging only. */
-    static struct freertos_addrinfo * pxLastInfo = NULL;
+    _static struct freertos_addrinfo * pxLastInfo = NULL;
```

Comment:
If this is only for debugging, then why is it always compiled in?  Shouldn't it be in something like:

```
#ifdef _FREERTOS_DNS_DEBUG_
_static struct freertos_addrinfo * pxLastInfo = NULL; 
#endif 
```

Reply:
Thanks for the review @gkwicker.
Since IPv6 is under development, this is something which is can be used a breakpoint to check the value.
While I agree that the variable should be conditionally compiled, it is later used in below format:
```
configASSERT( pxLastInfo != pxInfo );
        pxLastInfo = pxInfo;

        while( pxInfo != NULL )
        {
            pxNext = pxInfo->ai_next;
            vPortFree( pxInfo );
            pxInfo = pxNext;
        }

        pxLastInfo = NULL;
```

I think that if all instances are conditionally compiled using `#ifdef`, then it would make the code difficult to read and a bit messy. I believe that we can get rid of this (and all other debugging variables/prints) once this library has been thoroughly tested.

Or shall I use the `#ifdef`?

Reply:
Then the comment at line 380 is incorrect, `pxLastInfo` is _not_ used for debugging only; it is used to implement additional error checking via `configASSERT()`.  The correct thing to do here is to update the comment so that it is accurate, e.g.: _"Used for additional error checking when asserts are enabled."_

Reply:
Done!